platform-sdk/docs/proposals/001-tss-rosters-design-proposal/proposal.md

```diff
@@ -293,173 +313,157 @@ the state:
+A common example of this is Mainnet State being transplanted into a testing network.
 
-A quick note on PCES. The software version is used to determine which roster (active or previous) should be used to
-validate the signatures of events in the PCES. If the PCES event version is less than the current software version,
-it indicates that the event was signed with the old roster, and hence, the old roster is used for validation. If the
-PCES event version matches the current software version, the active roster is used. This behavior will be
-maintained until Birth Rounds or Dynamic Address Book is implemented.
+A quick note on Pre-Consensus Event Stream (PCES).
+The software version is used to determine which roster (active or previous) should be used to validate the signatures of events in the PCES.
+If the PCES event version is less than the current software version, it indicates that the event was signed with the old roster, and hence, the old roster is used for validation.
+If the PCES event version matches the current software version, the active roster is used.
+This behavior will be maintained until Birth Rounds or Dynamic Address Book is implemented.
 
-Therefore, Provided an existing State, an `Override Roster` and a network upgrade (which could be a simple
-config-only upgrade if a software version upgrade isn't required), the platform will adopt the provided Override Roster.
+Like the Genesis Roster, the Override Roster will be created from a special-purpose config file named `override-config.txt`.
+This file will be in the same location and format as the current `config.txt` file.
+
+Therefore, Provided an existing State, an `override-config.txt` and a network upgrade (which could be a simple config-only upgrade if a software version upgrade isn't required), an `Override Roster` will be constructed from the `override-config.txt` and adopted as the active `Roster`.
+Now might be a good time to revisit the startup decision tree diagram for reference.
 
 The next round number for this network will continue from the last round number in the provided state.
-The trigger for adopting the `Override Roster` will be the presence of an Override Roster at start-up with a State
-existing on disk. Bear in mind the Network Transplant process is designed primarily with test networks in mind.
-Upon adoption of the Override Roster, the Override Roster should be deleted from disk. In test networks where this
-process is designed to be used, the Override Roster can be easily accessed and deleted by DevOps.
+Note that the Network Transplant process is designed primarily with test networks in mind.
+Upon adoption of the `Override Roster`, the `override-config.txt` should be deleted/renamed from disk.
+In test networks where this process is designed to be used, the `override-config.txt` file can be easily accessed and deleted by DevOps.
```

Comment:
The wording here is a bit ambiguous. The first sentence says "it should be deleted". A reader would assume something will implement something to delete it.

Then the second sentence says "it can be deleted"... manually. Okay.

Then what happens if it could be deleted, but actually wasn't deleted? Does this cause an error? Is this idempotent? Or....?

I suggest to rephrase this to clarify the procedure and/or mechanism.

Reply:
good catch. Clarified.